Atomic/syscontainers.py

```diff
@@ -152,9 +172,26 @@ def _pull_image_to_ostree(self, repo, image, upgrade):
         return image
 
     def pull_image(self, image=None):
+        """
+        Public method for pulling an image from an external location into ostree.
+
+        :param image: Name of the image to pull. If not provided self.args.image is used.
+        :type image: str or None
+        """
+        # TODO: guard against image and self.args.image being None?
```

Comment:
apparently tests fail if this line is added

Reply:
Bah. It's because ``TODO`` is considered bad by the checks. I'll remove it.